components/template-mgt/org.wso2.carbon.identity.template.mgt/src/main/java/org/wso2/carbon/identity/template/mgt/TemplateManager.java

```diff
@@ -30,6 +30,7 @@
  *
  * @since 1.0.0
  */
+@Deprecated
```

Comment:
do not deprecate template mgt core classes. only deprecate template mgt endpoint.

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2996/commits/51d084a813c79c52636b52d39f06a834dcb09ed1